src/drt/test/gcTest.cpp

```diff
@@ -1128,6 +1128,32 @@ BOOST_AUTO_TEST_CASE(metal_width_via_map)
              Rect(100, 0, 200, 100));
 }
 
+BOOST_AUTO_TEST_CASE(cut_spc_parallel_overlap)
```

Comment:
Very good. Could you now make it a BOOST_DATA_TEST_CASE with two tests; one should have a violation marker and the other shouldn't?